unitOneAssessmentPractice.js

```diff
@@ -14,22 +16,43 @@ let assert = require('assert')
 // Uncomment out the next line to test your solution
 // runQ2Tests()
 
+const containsSeven = (num) => {
+  let stringToNum = num.toString()
+  return stringNum.includes('7')
```

Comment:
speaking of variable names - whatever you name them make sure you always write it the same way everywhere. You function works, if you correct the return to "stringToNum.includes("7")"